Formula/imgproxy.rb

```diff
@@ -39,14 +39,13 @@ def install
 
     output = testpath/"test-converted.png"
 
-    system "curl", "-s", "-o", output,
-           "http://127.0.0.1:#{port}/insecure/fit/100/100/no/0/plain/local:///test.jpg@png"
-    assert_equal 0, $CHILD_STATUS
+    safe_system "curl", "-s", "-o", output,
+                "http://127.0.0.1:#{port}/insecure/resize:fit:100:100:true/plain/local:///test.jpg@png"
```

Comment:
```suggestion
    system "curl", "-s", "-o", output,
           "http://127.0.0.1:#{port}/insecure/resize:fit:100:100:true/plain/local:///test.jpg@png"
```
There's no need to use `safe_sytem` in formulae. `Formula#system` is always `safe`

Reply:
Ah, right, thanks!
Then we just don't need to check exit code in this case